### PR TITLE
Allocation implementation v3 [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by ppalucki**

### CODE REVIEW

**Comment by ppalucki** [comment]
nobody uses this branch to for overview look for allocation

File: owca/runners/allocation.py [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+            # Some wanted unavailable feature - halt.
+            raise Exception("RDT MB control is not supported by platform!")
+        elif self._rdt_mb_control_enabled is None:
+            # Autoconfiguration of rdt mb control.
+            self._rdt_mb_control_enabled = platform.rdt_information.rdt_mb_control_enabled
+
+        root_rdt_l3, root_rdt_mb = get_max_rdt_values(platform.rdt_information.cbm_mask,
+                                                      platform.sockets)
+        # override max values with values from allocation configuration
+        if self._allocation_configuration.default_rdt_l3 is not None:
```
Comment (iwankgb) [inline owca/runners/allocation.py]: What happens when `self._allocation_configuration.default_rdt_l3 > root_rdt_l3`? It should not be allowed and I can't find any code validating it.

File: owca/runners/allocation.py [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+            root_rdt_l3 = self._allocation_configuration.default_rdt_l3
+        if self._allocation_configuration.default_rdt_mb is not None:
```
Comment (iwankgb) [inline owca/runners/allocation.py]: See my comment about validation (just above).

**Comment by ppalucki** [comment]
superseded by final #86

**Pull request closed without merging**